_layouts/syllabus.html

```diff
@@ -10,7 +10,7 @@
     <strong>{{ page.title | markdownify }}</strong>
     {% if page.lectureTopics %}
       <p>Lecture</p>
-      <ul>
+      <ul>        
```

Comment:
Ack! Extraneous whitespace!
(Seems picky, I know, but you're already gonna be making edits, so I don't feel as bad requesting to remove it)